src/plugins/cosmos/components/modals/Staking/views/Unstake.tsx

```diff
@@ -49,10 +49,17 @@ type UnstakeProps = {
   assetId: CAIP19
   cryptoAmountStaked: string
   marketData: MarketData
+  validatorAddress: string
 }
 
 // TODO: Wire up the whole component with staked data
```

Comment:
delete comment now?

Reply:
I guess it should be removed from https://github.com/shapeshift/web/pull/1365